panther_analysis_tool/rule.py

```diff
@@ -133,6 +139,18 @@ def __init__(self, config: Mapping):
         self.detection_id = config["id"]
         self.detection_version = config["versionId"]
 
+        # backwards compatible for legacy mocking
+        self._setup_exception = None
+        if "lastUpdatedVersion" in config:
+            try:
+                detection_panther_version = semver.VersionInfo.parse(config["lastUpdatedVersion"])
+            except Exception as err:  # pylint: disable=broad-except
+                self._setup_exception = err
```

Comment:
Hmm, this error seems to be Panther's responsibility though, so we should get notified for it, shouldn't we? If you store it as a setup exception it translates to the module not being able to be loaded or something relating to detection code.

Reply:
Yeah, I think you're right -- I'll remove the try-except block so the exception is raised